### ISSUE TITLE
Collection generator creates filenames with underscores

### ISSUE THREAD

**Issue by Rocketpilot** [galliani/jekyll-airtable#1]
When I construct a file name in a template using the `slugify` filter, phrases become hypenated-like-so, but the plugin creates filenames with underscores instead. Could this be a setting?

**Comment by galliani** [comment]
yes, ATM I set it up to use underscores instead by default. I have not used slugify at all so not familiar with it. What configuration do you prefer?

**Comment by Rocketpilot** [comment]
Slugify is just a Liquid string filter to allow template designers to turn arbitrary text strings into URL slugs (cf https://jekyllrb.com/docs/liquid/filters/). It's very handy, when, for example, you know in advance that the title of a post is going to be the URL slug and when the slug is not already available as a variable. 

It seems to default to hyphens rather than underscores, so that would be the preference.

Cheers!

**Comment by galliani** [comment]
I agree with you, I have included this approach in the v0.4.1 of this gem.
